src/controllers/comment.js

```diff
@@ -124,16 +124,63 @@ class Comment {
       });
 
       if (!postComment) {
-        return response.status(500).send(errorResponse(['Unable to post comment']));
+        return response.status(500).json(errorResponse(['Unable to post comment']));
       }
 
-      return response.status(201).send({
-        comment: {
-          body: commentBody
-        }
-      });
+      return response.status(201).json({ comment: { body: commentBody } });
+    } catch (error) {
+      return response.status(500).json(errorResponse(['Server error: Failed to post comment']));
+    }
+  }
+
+  /**
+   * @description - Update/Edit a comment by ID
+   *
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   *
+   * @returns {object} - object representing response message
+   *
+   * @memberof Comment
+   */
+  static async update(request, response) {
+    const { commentId = '' } = request.params;
+
+    if (!commentId) return response.status(400).json(errorResponse(['Please provide ID of comment to update']));
+    if (!parseInt(commentId, 10)) return response.status(400).json(errorResponse(['Please enter a valid comment ID']));
+
+    try {
+      const { payload } = request.userData;
+      const { id: userId } = payload;
+
+      const existingComment = await CommentModel.findByPk(commentId);
+
+      if (!existingComment) return response.status(404).json(errorResponse(['Comment not found']));
+      if (userId !== existingComment.userId) return response.status(401).json(errorResponse(['Not allowed']));
+
+      const { body: newComment } = request.body;
+
+      if (!newComment || !newComment.trim()) return response.status(400).json(errorResponse(['Comment body is required']));
+      if (newComment === existingComment.body) return response.status(200).json({ msg: 'Comment was not edited because content is the same' });
```

Comment:
The success response object should contain `message`, not `msg` per the team's convention.

![image](https://user-images.githubusercontent.com/15332525/50334128-9e48ec80-0507-11e9-887d-cce5d1eda6ac.png)